spec/requests/barcode_gets_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+require 'marc'
+
+RSpec.describe "Barcode Gets", :type => :request do
+  describe "GET /barcode/32101070300312" do
+    it "returns a collection of bibliographic records" do
+      stub_voyager_barcodes('32101070300312')
+      get '/barcode/32101070300312'
+      expect(response.status).to be(200)
+    end
+  end
+  describe "GET /barcode/321010702214" do
+    it "returns an error when the barcode is not a valid form" do
+      allow(VoyagerHelpers::Liberator).to receive(:get_records_from_barcode).and_return([])
+      get '/barcode/321010702214'
+      expect(response.status).to be(404)
+    end
+  end
+  describe "GET /barcode/32101070300319" do
```

Comment:
This test case is unnecessary because it's the same situation as the one above - when the `get_records_from_barcode` method returns and empty array the status should be 404.